### PR TITLE
[Bulky] Allow retrying of failed CSC payments.

### PR DESCRIPTION
**Pull request by dracos**
[skip changelog] Fixes https://github.com/mysociety/societyworks/issues/3781
The form is shown to staff at /waste, as requested by Peterborough.

### CODE REVIEW

File: templates/web/base/waste/garden/csc_payment_failed.html [hunk 1]
```diff
@@ -16,6 +13,13 @@ <h1 class="govuk-panel__title">
         <p>
         No subscription will be created.
```
Comment (davea) [inline templates/web/base/waste/garden/csc_payment_failed.html:14]: Might be confusing, should this be "No booking has been created"?

**Review by davea — CHANGES_REQUESTED** [review]
Entering the reference number on the address page works, but the "Retry booking" button on the property page seems to start the process from scratch with no details prefilled in the report form:

image

**Comment by dracos** [comment]
> Entering the reference number on the address page works, but the "Retry booking" button on the property page seems to start the process from scratch with no details prefilled in the report form:

Doh, I moved the form to the front page from bin days page as they confirmed that's where they wanted it but entirely forgot about the button on the bin days page, sorry!

**Review by davea — APPROVED** [review]
💰 ↩️ 🆗

**Pull request merged**